### PR TITLE
[APP-139/144] Updates request changes

### PR DESCRIPTION
**Pull request by phillipsgoncalves**
**What does this PR do?**

This PR aims at changing the updates logic. Now we are only making the updates request when the user navigates to the apps BN section. We also had to change the size of the updates badge.
Also took the chance to remove an unnecessary call.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] AppsManager.java

**How should this be manually tested?**

Open the app, check that there are no updates and the badge is still shown to hook the user. Then navigate to apps, the request will be made and it will show the updates number. On the next sessions, everything should be normal.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-139](https://aptoide.atlassian.net/browse/APP-139)
[APP-144](https://aptoide.atlassian.net/browse/APP-144)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

### CODE REVIEW

File: app/src/main/res/drawable/updates_badge_circle_small.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval">
+
+  <solid android:color="@color/red" />
+
+  <stroke
+      android:width="1dp"
+      android:color="@color/default_background_color_light" />
```
Comment (jdandrade) [inline app/src/main/res/drawable/updates_badge_circle_small.xml:9]: just checking.. will this be ok both themes?

**Review by jdandrade — CHANGES_REQUESTED** [review]

File: app/src/main/java/cm/aptoide/pt/home/apps/AppsManager.java [hunk 2]
```diff
@@ -65,13 +71,21 @@ public AppsManager(UpdatesManager updatesManager, InstallManager installManager,
...
   }
 
   public Observable<List<UpdateApp>> getUpdatesList() {
-    return Observable.combineLatest(getAllUpdatesList(), getUpdateDownloadsList(),
-        this::mergeUpdates);
+    if (SecurePreferences.isUpdatesFirstLoad(secureSharedPreferences)) {
+      return updatesManager.refreshUpdates()
+          .startWith(updateFirstLoadUpdatesSettings())
+          .andThen(startUpdatesNotification())
+          .andThen(getAllUpdatesList());
```
Comment (jdandrade) [inline app/src/main/java/cm/aptoide/pt/home/apps/AppsManager.java:84]: in terms of performance, is the loading of the apps view ok? time?

File: app/src/main/res/drawable/updates_badge_circle_small.xml (hunk 1 above)
Comment (phillipsgoncalves) [reply app/src/main/res/drawable/updates_badge_circle_small.xml:9]: yes, i created a different drawable under the drawable-night folder

File: app/src/main/java/cm/aptoide/pt/home/apps/AppsManager.java (hunk 2 above)
Comment (phillipsgoncalves) [reply app/src/main/java/cm/aptoide/pt/home/apps/AppsManager.java:84]: yes, i didn't notice that much of an issue, eventhough i have 37 updates on my xiaomi for example (and a lot of apps installed). It takes a bit longer than usual, but it is normal as it is making the request (but still under 2 seconds) and obviously with this solution we will alwyas be dependent on our services response times.

**Review by jdandrade — APPROVED** [review]
lgtm

**Pull request merged**